smithy-typescript-codegen/src/test/java/software/amazon/smithy/typescript/codegen/StructureGeneratorTest.java

```diff
@@ -521,6 +533,7 @@ private String testStructureCodegen(String file, String expectedType) {
                         .withMember("service", Node.from("smithy.example#Example"))
                         .withMember("package", Node.from("example"))
                         .withMember("packageVersion", Node.from("1.0.0"))
+                        .withMember("backwardCompatibleRequiredMember", Node.from(backwardCompatibleRequiredMember))
```

Comment:
Echoing above, with this naming, `backwardCompatibleRequiredMember`, it's hard to understand what this will change in the generated code.